ginivision/src/main/java/net/gini/android/vision/noresults/package-info.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * <p>
+ * Contains the Activity and Fragments used for the No Results Screen. The No Results Screen
+ * includes
+ * hints how to take a picture in an optimal condition. This screen should be shown when the
+ * received results from the Gini API don't include the required extractions.
+ * </p>
+ *
+ * <h3>Screen API</h3>
+ *
+ * <p>
+ * The {@link net.gini.android.vision.noresults.NoResultsActivity} should be shown when there are
```

Comment:
We should formulate this from the clients' perspective. The NoResultsActivity is launched for them by the AnalysisActivity so we should inform them about this and the conditions for showing, because they won't have to launch it themselves.

The OnboardingActivity is a bit similar as it's launched in the Screen API by the CameraActivity: https://github.com/gini/gini-vision-lib-android/blob/9d3827a45f318a7dd45c93c80ad934f39ae9a8a4/ginivision/src/main/java/net/gini/android/vision/onboarding/package-info.java#L9